contracts/malicious/malicious.hpp

```diff
@@ -0,0 +1,113 @@
+/**
+ *  @file
+ *  @copyright defined in eos/LICENSE.txt
+ */
+
+#include <eoslib/eos.hpp>
+#include <eoslib/token.hpp>
+#include <eoslib/db.hpp>
+
+/**
+ *  @defgroup examplecontract Example Contract
+ *  @brief Example Contract
+ *  @ingroup contractapi
+ *
+ */
+
+
+/**
+ * Make it easy to change the account name the currency is deployed to.
+ */
+#ifndef TOKEN_NAME
+#define TOKEN_NAME currency
+#endif
```

Comment:
Naming this currency is confusing. This has nothing to do with TOKENs. Please remove use of TOKEN_NAME.